style.css

```diff
@@ -303,3 +303,78 @@ body {
   font-size: 15px;
   color: #253858;
 }
+
+.contact-section {
+  background-color: #fff;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.contact-form-section {
+  background-color: #6070ff;
+  padding: 0;
+  border-top-left-radius: 125px;
+  width: 100%;
+  box-sizing: border-box;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 50px;
+  justify-items: center;
+}
```

Comment:
As I've mentioned earlier, you guys have done an amazing job. Your contact form works well and the UI is great :muscle: 

I realized that you've used the `Grid` layout style for this section. The grid layout works fine and it's acceptable. However, this project specifically stated that you should use CSS Flexbox layout. It is important we always stick to the project specification. Therefore, I suggest you replace the grid layout with flexbox.